docs/smart-contracts/creator-tools/rewards.mdx

```diff
@@ -33,20 +33,33 @@ Rewards v1.1 is deployed at the same address on all networks.
 | First Minter    | 0.000111 ETH |
 | Zora            | 0.000111 ETH |
 
-#### Paid mint
+#### Paid Mint
 
 | Recipient       | Amount       |
 | --------------- | ------------ |
+| Creator | 100% of earnings from sales |
 | Create Referral | 0.000222 ETH |
 | Mint Referral   | 0.000222 ETH |
 | First Minter    | 0.000111 ETH |
 | Zora            | 0.000222 ETH |
 
+#### ERC20 Mint
```

Comment:
is this duplicated in the erc20 minter section? if so, then that should link to here and we should remove it from there, so that we don't have to maintain these 2 tables